editor/editor_node.cpp

```diff
@@ -3003,6 +2984,44 @@ void EditorNode::_tool_menu_option(int p_idx) {
 	}
 }
 
+void EditorNode::_convert_to_menu_option(int p_idx) {
+	switch (convert_to_menu->get_item_id(p_idx)) {
+		case FILE_EXPORT_MESH_LIBRARY: {
+			current_option = FILE_EXPORT_MESH_LIBRARY;
+
+			if (!editor_data.get_edited_scene_root()) {
+				show_accept(TTR("This operation can't be done without a scene."), TTR("OK"));
+				break;
+			}
+
+			List<String> extensions;
+			Ref<MeshLibrary> ml(memnew(MeshLibrary));
+			ResourceSaver::get_recognized_extensions(ml, &extensions);
+			file_export_lib->clear_filters();
+			for (const String &E : extensions) {
+				file_export_lib->add_filter("*." + E);
+			}
+
+			file_export_lib->popup_file_dialog();
+			file_export_lib->set_title(TTR("Export Mesh Library"));
+
+		} break;
+		case CONVERT_TO_CUSTOM: {
+			if (convert_to_menu->get_item_submenu(p_idx).is_empty()) {
```

Comment:
Is submenu checked to be empty correct? Will test.

Reply:
```suggestion
			if (!convert_to_menu->get_item_submenu(p_idx).is_empty()) {
```

Can you check this logic and reply back?

Reply:
If the submenu is not empty, we don't need to trigger the callback. Only the submenu's PopupMenu needs to displayed (which the submenu will take care of by itself).

Reply:
Thanks.